docs/1.0. Certificate Provisioning.md

```diff
@@ -221,126 +225,128 @@ If an EST Client implements the AMWA IS-04 Node API, then the EST Client SHOULD
+1. The EST Client MAY have a TLS Certificate for the target network manually installed on the device. The manually installed TLS certificate MUST then be used to secure its NMOS APIs until the TLS Certificate is due for renewal. The manually installed TLS Certificate MUST be presented during the TLS Handshake for [Certificate Renewals](#certificate-renewal). If the certificate renewal is successful, the manually installed TLS Certificate and private key MUST be removed.
+2. The manufacturer-issued TLS Client Certificate MAY be renewed by a software/firmware update; this update MUST contain a unique TLS certificate per device. The valid TLS Client Certificate MAY then be used during [Initial Certificate Provisioning](#initial-certificate-provisioning).
 
-### Connection to new network
+### Connection to New Network
 
 It MAY be desirable that when a EST Client is connected to a different network that it automatically requests a TLS Certificate for the new network.
 
-On start up or on change of network state the EST Client MUST attempt to discover the EST Server using [DNS-SD](#dns-sd-advertisement), unless the location of the EST Server is manually configured. The EST Client SHOULD make a request to the `/cacerts` endpoint, if the request is successful the EST Client SHOULD compare the returned Certificate to the currently install Root CA. If the Certificate is for a different Domain the EST Client MUST follow the [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow.
+On start up or on change of network state, the EST Client MUST attempt to discover the EST Server using [DNS-SD](#dns-sd-advertisement), unless the location of the EST Server is manually configured. The EST Client SHOULD make a request to the `/cacerts` endpoint. If the request is successful, the EST Client SHOULD compare the returned Certificate to the currently installed Root CA. If the Certificate is for a different domain, the EST Client MUST follow the [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow.
 
 ### Certificate Revocation
 
-An EST Client SHOULD periodically check the revocation status of both the Root CA and their TLS Certificates using [OCSP][RFC-6960] and [CRL][RFC-5280]. If a TLS Certificate is revoked, the EST Client MUST stop using the revoked certificate immediately and follow [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow to replace the certificate.
+An EST Client SHOULD periodically check the revocation status of both the Root CA and their(its?) TLS Certificates using [OCSP][RFC-6960] and [CRL][RFC-5280]. If a TLS Certificate is revoked, the EST Client MUST stop using the revoked certificate immediately and follow [Initial Certificate Provisioning](#initial-certificate-provisioning) workflow to replace the certificate.
 
-## Notes to Implementers
+## Notes(?) to Implementers
+(MOVE TO IMPLEMENTATION GUIDE?)
```

Comment:
"Notes to implementers" shouldn't use normative language, should it?
So if it's  normative, rename the section. If it's not, remove from here, and move to INFO-002 in that doc's next edition.

Reply:
Agreed. This section appears to include both actual normative items and items that look normative but I don't think are e.g. "...MAY not be valid"